### PR TITLE
Attach EC2 Security Groups to RDS Instances

### PR DESCRIPTION
**Pull request by achantavy**

### CODE REVIEW

File: cartography/intel/aws/rds.py [hunk 1]
```diff
@@ -105,9 +105,33 @@ def load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_
...
+        _attach_ec2_security_groups(neo4j_session, rds, aws_update_tag)
     _attach_read_replicas(neo4j_session, read_replicas, aws_update_tag)
 
 
+def _attach_ec2_security_groups(neo4j_session, instance, aws_update_tag):
+    """
+    Attach an RDS instance to its EC2SecurityGroups
+    """
+    attach_rds_to_group = """
+    MATCH (rds:RDSInstance{id:{RdsArn}}),
```
Comment (sachafaust) [inline cartography/intel/aws/rds.py]: I know we have ARN as ID but we could benefit from having this simpler for our OSS users and use the ARN attribute (assuming it's indexed)

File: cartography/intel/aws/rds.py [hunk 2]
```diff
@@ -105,9 +105,33 @@ def load_rds_instances(neo4j_session, data, region, current_aws_account_id, aws_
...
+    (sg:EC2SecurityGroup{groupid:{GroupId}})
+    MERGE (rds)-[m:MEMBER_OF_EC2_SECURITY_GROUP]->(sg)
```
Comment (sachafaust) [inline cartography/intel/aws/rds.py]: BLOCK - missing cleanup job?

**Comment by sachafaust** [comment]
added blocking comment

File: cartography/intel/aws/rds.py (hunk 1 above)
Comment (achantavy) [reply cartography/intel/aws/rds.py]: See https://github.com/lyft/cartography/blob/master/cartography/data/indexes.cypher#L40.

We also index on RDS' db_instance_identifier which is more of a friendly name.  I can make it clearer in the schema docs that this is the case.

File: cartography/intel/aws/rds.py (hunk 2 above)
Comment (achantavy) [reply cartography/intel/aws/rds.py]: We clean up RDS instances [here](https://github.com/lyft/cartography/blob/master/cartography/intel/aws/rds.py#L98) (actual job is [here](https://github.com/lyft/cartography/blob/master/cartography/data/jobs/cleanup/aws_import_rds_instances_cleanup.json) and EC2 Security Groups [here](https://github.com/lyft/cartography/blob/master/cartography/intel/aws/ec2.py#L583) (actual job is [here](https://github.com/lyft/cartography/blob/master/cartography/data/jobs/cleanup/aws_import_ec2_security_groupinfo_cleanup.json).

Comment (sachafaust) [reply cartography/intel/aws/rds.py]: What about MEMBER_OF_EC2_SECURITY_GROUP?

Comment (achantavy) [reply cartography/intel/aws/rds.py]: [`detach delete`](https://neo4j.com/docs/cypher-manual/current/clauses/delete/#query-delete-introduction) removes the relationships as well.

Comment (sachafaust) [reply cartography/intel/aws/rds.py]: What if both rds and security group are still there but they aren't associated anymore?

Comment (achantavy) [reply cartography/intel/aws/rds.py]: Ok, we can add a statement to do this.

Comment (achantavy) [reply cartography/intel/aws/rds.py]: Added more to the cleanup job, how's this look now?

Comment (sachafaust) [reply cartography/intel/aws/rds.py]: LGTM

**Pull request merged**